minorminer/utils/raster_embedding.py

```diff
@@ -320,6 +402,7 @@ def embeddings_to_ndarray(embs, node_order=None):
         value_list = [v for emb in embs for v in emb.values()]
         assert len(set(value_list)) == len(value_list)
 
+        visualize_embeddings(T, embeddings=embs, title=f'{stopology.capitalize()} Embedding')
```

Comment:
remove `title` (btw, note `stopology` is re-using values from the previous loop)